.github/workflows/ci.yml

```
@@ -155,10 +155,21 @@ jobs:
         shell: bash
         run: echo ::set-output name=flavor::$([[ "${{ matrix.os }}" == "cuda" ]] && echo "Cuda" || echo "CPU")
 
+      - name: Set values
+        id: values
+        shell: bash
+        run: |
+          if [[ ${{ github.event_name == 'schedule' || (github.event_name == 'push' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v'))) }} ]]; then
+            jsonValues="{"ILGPU_CLEAN_TESTS":"1", "ILGPU_TEST_PARALLELIZATION_LEVEL":"true"}"
```

ISSUE: It looks like JSON may be incorrect. Clean tests should be a boolean, while parallelisation level should be a number.
```suggestion
            jsonValues="{"ILGPU_CLEAN_TESTS":"true", "ILGPU_TEST_PARALLELIZATION_LEVEL":"1"}"
```

---

Thanks, @MoFtZ !